keyboards/soy20/keymaps/default/keymap.c

```diff
@@ -0,0 +1,77 @@
+#include "soy20.h"
```

Comment:
```suggestion
#include QMK_KEYBOARD_H
```

The more generic header is preferred here.